CRM/Core/Payment/Form.php

```diff
@@ -290,6 +290,15 @@ public static function setDefaultValues(&$form, $contactID) {
       $form->_defaults["billing_state_province_id-{$form->_bltID}"] = CRM_Core_Config::singleton()
         ->defaultContactStateProvince;
     }
+
+    // Set default payment processor
+    if (!empty($form->_paymentProcessors)) {
```

Comment:
Ah - making it public so a static function can alter it - hmm - maybe we can set this up in a trait or something instead